src/Auth0.Core/ApiError.cs

```diff
@@ -27,10 +28,23 @@ public class ApiError
         [JsonProperty("message")]
         public string Message { get; set; }
 
-        /// <summary>
-        /// The HTTP Status code.
-        /// </summary>
-        [JsonProperty("statusCode")]
-        public int StatusCode { get; set; }
+        public static ApiError ParseOrDefault(string responseContent)
+        {
+            if (String.IsNullOrEmpty(responseContent))
+                return default;
+
+            try
+            {
+                return JsonConvert.DeserializeObject<ApiError>(responseContent);
+            }
+            catch (Exception)
+            {
+                return new ApiError
+                {
+                    Error = responseContent,
```

Comment:
Does `Error` represent an error code? If so, I would prefer `unknown` or thereabouts instead of a duplicate of the message.

Reply:
That's what was already here - just moved around.  We could break it if you think there's value.

Reply:
I'm just thinking that duplicating the content in both is unnecessary. An unset `Error` would be more helpful than the same as `Message`

Reply:
Will figure out a better pattern for this in another PR.